tsl/test/isolation/specs/reorder_vs_select.spec

```diff
@@ -14,7 +14,7 @@ setup {
      index REGCLASS=NULL,
      verbose BOOLEAN=FALSE,
      wait_on REGCLASS=NULL
- ) RETURNS VOID AS '$libdir/timescaledb-1.2.0-dev', 'ts_reorder_chunk' LANGUAGE C VOLATILE;
+ ) RETURNS VOID AS '$libdir/timescaledb-1.3.0-dev', 'ts_reorder_chunk' LANGUAGE C VOLATILE;
```

Comment:
Is it possible to modify this so that we don't refer to specific versions of the library in the script? Maybe a var like $libdir/$libversion?  That should help switch between libraries without changing these files. I know the hard-coded name was not introduced here. But wondering if it can be addressed here and in the other .spec files.

Reply:
@gayyappan these files aren't really hardcoded, but are generated by CMake during the build. They were never supposed to be tracked by git in the first place but during the great rename from `recluster` to `reorder` the `.gitignore` was missed.  
I part of the fix was merged in [PR 1033](https://github.com/timescale/timescaledb/pull/1033/files#diff-10a0179798ffccfce8fa10934b3f576b) and I'll have the rest of it up soon.  
Unfortunately, CMake generation seems to be the best we can do at the moment; `$libdir` isn't actually a shell variable, but a magic string understood by `load_file`.

Reply:
ok, thanks for the explanation.

Reply:
Fix also in this PR: https://github.com/timescale/timescaledb/pull/1039